docs/monoDrive_home/Vehicle-Configuration.md

```diff
@@ -1,89 +1,213 @@
-# Vehicles
+# Scenario Vehicle
+The monoDrive Simulator provides users with the ability to customize the color and model to use for the ego vehicle as well as for the vehicles that interact with it on simulation. The simulator also provides users with the ability to specify the initial conditions for any car on the scene, i.e. initial pose and initial velocity. 
 
-## Vehicle
+## Change the vehicle's model
+<div class="img_container">
+  <video width=650px height=480px muted controls autoplay loop>
+    <source src="http://cdn.monodrive.io/readthedocs/vehicles.mp4" type="video/mp4">
+  </video>
+</div>
+**Current Blueprints available**   
+On the monoDrive Editor look for the Content Browser, from the folders select Vehicles. You will find the current models available.
+
+When selecting a model programmatically (i.e. using any client) you can assign the model to use for the ego vehicle to be random from the vehicle blueprints, you can find it on your Editor folder under:  
+`Config/ConfigRandomVehicleSettings.ini`   
+
+```
+[VehicleBlueprints]
+VehicleBlueprint=Blueprint'/Game/Vehicles/sedan_monoDrive_01.sedan_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/sedan_monoDrive_02.sedan_monoDrive_02_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/SUV_monoDrive_01.SUV_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/truck_monoDrive_01.truck_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/subcompact_monoDrive_01.subcompact_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/minivan_monoDrive_01.minivan_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/coupe_monoDrive_01.coupe_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/crossover_monoDrive_01.crossover_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/compact_monoDrive_01.compact_monoDrive_01_C'
+```
+**Note:** If you want to add your own vehicle model, add the path to the blueprint to this file.  
```

Comment:
Leaving out "you", try: To add a customized vehicle model, add the path to the blueprint in this file.